### DISCUSSION TITLE
Package structure of HomeCare/Disaster survey

### DISCUSSION THREAD

**Discussion by shunkosa** [SFDO-Community-Sprints/GrassrootsMobileSurveyApp#63] [General]
Currently, our Salesforce package contains the minimum of fields and page layouts used in mobile applications. Should we contain HomeCare and Disaster fields, layouts and record types in the same package? If some users don't need the example fields, it might be better to separate the plain base package (current ver) and the package containing example metadata into different installation URLs. Any thoughts?

**Comment by mjh-jhm** [comment]
Hi @shunkosa by default, Metecho will push the change to force-app or to a defined source directory.
I am for separating the sample survey metadata from the main package since it is a just sample and eventually gets deleted. We will have to update the installation instruction for the one with the sample project directory. I will bundle them together tomorrow with a permission set to the survey object and dependencies and push them to Metecho/sample_surveys branch.
Let's hear what others think as well.
 Thank you

**Comment by kdveverka** [comment]
Just updating this to note that we have decided to have the sample survey package as a seperate, optional item for folks
